vendors.json

```diff
@@ -73,6 +73,12 @@
     "constant": "DSARI",
     "env": "DSARI"
   },
+  {
+    "name": "GitHub Actions",
+    "constant": "GITHUB_ACTIONS",
+    "env": "GITHUB_ACTIONS",
```

Comment:
I believe this should be singular `GITHUB_ACTION`

https://help.github.com/en/articles/virtual-environments-for-github-actions#default-environment-variables

Reply:
No, `GITHUB_ACTION` is set to the name of the action that’s running (if there is one).  For example, `action/setup-node@v1`.

`GITHUB_ACTIONS` is set always.  I’ll put in a PR to the docs to reflect this.

Reply:
I'm not seeing the `GITHUB_ACTIONS` (plural) referenced anywhere. Can you share a link to that documentation?

Reply:
Sorry about the delay, we don't have the documentation update live yet.

Reply:
GITHUB_ACTIONS is [now documented here](https://help.github.com/en/github/automating-your-workflow-with-github-actions/using-environment-variables#default-environment-variables)